packages/client/wallets/aa/src/adapters/reservoir.ts

```diff
@@ -28,8 +29,7 @@ export function reservoirAdapter(aaWallet: EVMAAWallet): ReservoirWallet {
         },
         handleSendTransactionStep: async (chainId, stepItem, _) => {
             const stepData = stepItem.data;
-            return await aaWallet.sendTransaction({
-                chainId: chainId,
```

Comment:
Not sure, but I think I saw issues where I had to include this–any proof to being able to remove or just saw that it's optional?
Test cases here to confirm would be to use with:
- amoy
- optimism
- astar (if supported by reservoir, I could see this being different)